pkgs/applications/display-managers/greetd/qtgreet.nix

```
@@ -0,0 +1,62 @@
+{ stdenv
+, lib
+, fetchFromGitLab
+, pkg-config
+, meson
+, ninja
+, qt6
+, qt6Packages
```

```suggestion
, qtbase
, qttools
...
, dfl-utils
```
same for nativeBuildInputs/buildInputs

---

thanks for the tip!